Description: Upstream changes introduced in version 20110901-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 tpfand-profiles (20110901-1) natty; urgency=low
 .
   * Initial release of the Tpfanco fork
   * Add modified profile for Thinkpad Z61m
 .
 The person named in the Author field signed this changelog entry.
Author: Vladyslav Shtabovenko <DFEW.Entwickler@googlemail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- tpfand-profiles-20110901.orig/Makefile
+++ tpfand-profiles-20110901/Makefile
@@ -1,15 +1,18 @@
 DESTDIR=/
 
-install: all
-	install -d $(DESTDIR)/usr/share/tpfand/models/
-	install -d $(DESTDIR)/usr/share/tpfand/models/by-id	
+all:
+
+clean:
+
+install: all	
+	install -d $(DESTDIR)/usr/share/tpfand/models
+	install -d $(DESTDIR)/usr/share/tpfand/models/by-id
 	install -m 644 models/by-id/* $(DESTDIR)/usr/share/tpfand/models/by-id
-	install -d $(DESTDIR)/usr/share/tpfand/models/by-name
-	install -m 644 models/by-name/* $(DESTDIR)/usr/share/tpfand/models/by-name
+	install -d $(DESTDIR)/usr/share/tpfand/models/by-name	
 	echo Installation complete.	
 
-uninstall:	
-	rm -rf $(DESTDIR)/usr/share/tpfand/models/
+uninstall:
+	rm -rf $(DESTDIR)/usr/share/tpfand/models
 
 
 
--- tpfand-profiles-20110901.orig/models/by-id/lenovo_9450h9g
+++ tpfand-profiles-20110901/models/by-id/lenovo_9450h9g
@@ -1,20 +1,14 @@
-# Fan profile for tpfand
-# submitted on  2008-04-15 23:06:17
-# by an anonymous submitter
-# approved on  2008-04-19 14:50:43
-# downloaded on  2009-03-09 00:11:52.208303
-#
-0. Sensor 0 = 0:0 50:2 60:4 55:3 
-1. Sensor 1 = 0:0 40:2 50:4 45:3 
-2. Sensor 2 = 0:0 40:2 50:4 45:3 
+0. Sensor 0 = 0:0 55:3 60:255 
+1. Sensor 1 = 0:0 55:2 60:255 
+2. Sensor 2 = 0:0 40:2 45:3 50:255 
 3. Sensor 3 = 0:255 
-4. Sensor 4 = 0:0 40:2 45:3 
+4. Sensor 4 = 0:0 40:2 45:255 
 5. Sensor 5 = 0:255 
-6. Sensor 6 = 0:0 35:2 45:3 
+6. Sensor 6 = 0:0 40:2 45:255 
 7. Sensor 7 = 0:255 
-8. Sensor 8 = 0:0 40:2 45:3 
-9. Sensor 9 = 0:0 50:1 
-10. Sensor 10 = 0:0 55:1 
+8. Sensor 8 = 0:0 50:2 55:255 
+9. Sensor 9 = 0:0 55:2 60:255 
+10. Sensor 10 = 0:0 56:3 60:6 65:255 
 11. Sensor 11 = 0:255 
 12. Sensor 12 = 0:255 
 13. Sensor 13 = 0:255 
@@ -25,6 +19,3 @@ hysteresis = 2
 interval_speed = 2
 interval_duration = 500.000000
 interval_delay = 5000.000000
-
-comment = may need a little improvement still. but works like a charm at the moment in early-spring-europe. be careful in warmer countries. (submitted anonymously)
-
